add_product_susedata

```diff
@@ -337,13 +338,23 @@ for my $language (sort(@languages)) {
     my $pkgid = (grep {$_->{'pkgid'}} @{$pack->{'checksum'}})[0]->{'_content'};
     my $summary     = $pack->{'summary'}[0]->{'_content'};
     my $description = $pack->{'description'}[0]->{'_content'};
+    my $category;
+    if ($pack->{'format'}->{'rpm:provides'}->{'rpm:entry'}) {
+      for $p (@{$pack->{'format'}->{'rpm:provides'}->{'rpm:entry'}}) {
+          next unless $p->{'name'} eq 'pattern-category()';
+          $category = uri_unescape($p->{'ver'});
+          break; # we expect only one category
+      }
+    }
     my $i18n_summary     = gettext($summary);
     my $i18n_description = gettext($description);
+    my $i18n_category    = gettext($category) if $category;
```

Comment:
This is wrong, you must not use a trailing "if" with a my statement. It does not do what you expect.